src/util/handlebars/helpers.ts

```diff
@@ -1,9 +1,9 @@
 import hbs from "hbs";
 
 export const registerHandlebarsHelpers = () => {
-	hbs.registerHelper("toLowerCase", (str) => str.toLowerCase());
+	hbs.registerHelper("toLowerCase", (str?: string) => str?.toString().toLowerCase());
 
-	hbs.registerHelper("replaceSpaceWithHyphen", (str) => str.replace(/ /g, "-"));
+	hbs.registerHelper("replaceSpaceWithHyphen", (str?: string) => str?.toString().replace(/ /g, "-"));
```

Comment:
Where its being used and how can we test it?

If str is undefined, we'll get "Undefined', should we do an empty string instead?

Reply:
In `jira-configuration.hbs`.  We can add tests to it.  Good point on the empty string.

Reply:
Tests added.